src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java

```diff
@@ -0,0 +1,128 @@
+    private final JacksonMapper mapper;
+
+    public TradPlusBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequest) {
+        try {
+            final ExtImpTradPlus extImpTradPlus = parseImpExt(bidRequest.getImp().getFirst().getExt());
+            validateImpExt(extImpTradPlus);
+            HttpRequest<BidRequest> httpRequest = makeHttpRequest(extImpTradPlus, bidRequest.getImp(), bidRequest);
+            return Result.withValue(httpRequest);
+        } catch (PreBidException e) {
+            return Result.withError(BidderError.badInput(e.getMessage()));
+        }
+    }
+
+    private ExtImpTradPlus parseImpExt(ObjectNode extNode) {
+        final ExtImpTradPlus extImpTradPlus;
+        try {
+            extImpTradPlus = mapper.mapper().convertValue(extNode, EXT_TYPE_REFERENCE).getBidder();
+            return extImpTradPlus;
+        } catch (IllegalArgumentException e) {
+            throw new PreBidException(e.getMessage());
+        }
+    }
+
+    private void validateImpExt(ExtImpTradPlus extImpTradPlus) {
+        if (StringUtils.isBlank(extImpTradPlus.getAccountId())) {
+            throw new PreBidException("Invalid/Missing AccountID");
+        }
+    }
+
+    private HttpRequest<BidRequest> makeHttpRequest(ExtImpTradPlus extImpTradPlus, List<Imp> imps, BidRequest bidRequest) {
+        final String uri = endpointUrl.replace(ZONE_ID, extImpTradPlus.getZoneId()).replace(ACCOUNT_ID, extImpTradPlus.getAccountId());
+
+        final BidRequest outgoingRequest = bidRequest.toBuilder().imp(removeImpsExt(imps)).build();
+
+        return BidderUtil.defaultRequest(outgoingRequest, makeHeaders(), uri, mapper);
+    }
+
+    private MultiMap makeHeaders() {
+        return HttpUtil.headers().set(HttpUtil.X_OPENRTB_VERSION_HEADER, X_OPENRTB_VERSION);
+    }
+
+    private static List<Imp> removeImpsExt(List<Imp> imps) {
+        return imps.stream().map(imp -> imp.toBuilder().ext(null).build()).toList();
+    }
+
+    @Override
+    public Result<List<BidderBid>> makeBids(BidderCall<BidRequest> httpCall, BidRequest bidRequest) {
+        try {
+            final BidResponse bidResponse = mapper.decodeValue(httpCall.getResponse().getBody(), BidResponse.class);
+            return Result.withValues(extractBids(bidResponse, httpCall.getRequest().getPayload()));
+        } catch (DecodeException | PreBidException e) {
+            return Result.withError(BidderError.badServerResponse(e.getMessage()));
+        }
+    }
+
+    private static List<BidderBid> extractBids(BidResponse bidResponse, BidRequest bidRequest) {
+        return bidResponse == null || CollectionUtils.isEmpty(bidResponse.getSeatbid()) ? Collections.emptyList() : bidsFromResponse(bidResponse, bidRequest.getImp());
```

Comment:
I suspect this line fails checkstyle, please run it locally to check

Reply:
```
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:27:8: Unused import - java.util.ArrayList. [UnusedImports]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:56:37: Variable 'httpRequest' should be declared final. [FinalLocalVariable]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:79: Line is longer than 120 characters (found 123). [LineLength]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:80: Line is longer than 120 characters (found 135). [LineLength]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:106: Line is longer than 120 characters (found 167). [LineLength]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:110: Line is longer than 120 characters (found 243). [LineLength]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/main/java/org/prebid/server/bidder/tradplus/TradPlusBidder.java:125: Line is longer than 120 characters (found 134). [LineLength]
Error:  /home/runner/work/prebid-server-java/prebid-server-java/src/test/java/org/prebid/server/bidder/tradplus/TradPlusBidderTest.java:324: Line is longer than 120 characters (found 131). [LineLength]
```

Reply:
done.